mm2src/lp_ordermatch.rs

```diff
@@ -247,13 +257,15 @@ async fn process_orders_keep_alive(
 
     let mut orderbook = ordermatch_ctx.orderbook.lock();
     for (pair, diff) in response.pair_orders_diff {
+        let params = ProcessTrieParams {
+            pubkey: from_pubkey.clone(),
+            alb_pair: pair,
+            protocol_infos: response.protocol_infos.clone(),
```

Comment:
It borrowed before and now there's `clone`. Please pay more attention to this - I see unnecessary `clone` too often.

Reply:
Done